src/app/files/file-explorer.js

```diff
@@ -601,14 +605,24 @@ fileExplorer.prototype.createNewFile = function (parentFolder) {
 
 fileExplorer.prototype.createNewFolder = function (parentFolder) {
   let self = this
-  modalDialogCustom.prompt('Create new folder', '', '', (input) => {
+  modalDialogCustom.prompt('Create new folder', '', 'New folder', (input) => {
+    if (input === '') {
+      tooltip('Failed to create folder. The name can not be empty')
+      return false
```

Comment:
If you're using `return` to stop the process of the scope I would rath do `return tooltip(...)`. Returning `false` feels like it's used somewhere. If this is the case then forget this comment.